Tech and system model figures (#554)
NREL/GreenHEART: docs/intro.md

Depending on the models used and the size of the system, H2Integrate can simulate systems ranging from the kW to GW scale in seconds on a personal computer.
Additionally, H2Integrate tracks the flow of electricity, molecules (e.g., hydrogen, ammonia, methanol), and other products (e.g., steel) between different technologies in the energy system.

> H2Integrate uses a set of models to simulate the control, performance, cost, and finance of each technology, where only the performance and cost are required for most technologies.

Reviewer: ```suggestion
For each technology there are 4 different types of models: control, performance, cost, and finance. These model categories allow for modular pieces to be brought in or re-used throughout H2Integrate, as well as ease  of development and organization. Note that the only required models for a technology are performance and cost, while control and finance are optional. The figure below shows these four categories and some of the technologies included in H2Integrate. For a full list of models available, please see [Model Overview](user_guide/model_overview.md).
```

![A representation of a single technology model in H2Integrate](tech-model.png)

> The individual technology models are then connected to create the hybrid system model, as shown in the simplistic example below.

Reviewer: ```suggestion
The individual technology models are then connected to create the hybrid system model, as shown in the simplistic example below. Here, data from the performance, cost, and finance models of the grid and battery technologies feed into the overall system performance and finance calculations. There is also a physical connection between the grid and battery performance models in the form of an electrical cable. Lastly, within the battery technology, the control model and performance models are connected for dispatching of electricity.
```

![A representation of a system model in H2Integrate](system-model.png)

> If technologies require resource or price profiles, they can be provided by the user, or in many cases pulled automatically from existing databases. The costs and performance of the technology models in the system are combined for the system performance and system finance components for techno-economic analysis of the hybrid system. H2Integrate systems may include multiple system-level finance models to asses results with different system boundaries if desired. Besides simulation and analysis, H2Integrate can also perform system and sub-system optimization.

Author: corrected

> A more complex example of an H2Integrate model is shown below:

Reviewer: ```suggestion
A more complex and generalized example of an H2Integrate model is shown below:
```

![H2Integrate Splash Image](./splash_image.png)

> The modular nature of the H2Integrate system also makes adding custom models, including proprietary models for local analysis only, very straightforward.

Reviewer: ```suggestion
The modular nature of the H2Integrate system makes adding custom models, including proprietary models for local analysis only, very straightforward.
```

> ![H2Integrate Splash Image](./splash_image.png)

Author: I prefer leaving out the prefix if it is unnecessary.
